lib/chef/resource/windows_certificate.rb

```diff
@@ -79,9 +80,6 @@ class WindowsCertificate < Chef::Resource
         description: "Use the `CurrentUser` store instead of the default `LocalMachine` store. Note: Prior to #{ChefUtils::Dist::Infra::CLIENT}. 16.10 this property was ignored.",
         default: false
 
-      property :cert_path, String,
```

Comment:
If this is being replaced with output path we need to do it in a backward-compatible way. https://docs.chef.io/dsl_custom_resource/#deprecate-and-alias